assets/settings/default.json

```diff
@@ -328,6 +328,10 @@
   "title_bar": {
     // Whether to show the branch icon beside branch switcher in the titlebar.
     "show_branch_icon": false,
+    // Whether to show the branch button with the branch name in the titlebar.
+    "show_branch_button": true,
```

Comment:
maybe call this `show_branch_name` instead?

Reply:
Sure, thanks for checking.